src/Data/MongoDBMock/contests.js

```diff
@@ -22,36 +64,30 @@ const contests = [
     obedienceClasses: {
       0: [
         {
-          dogId: 'VIII-40407',
           dogName: 'Woof',
-          competingPairsId: '22-05-01Woof',
+          summaryId: '22-05-01Woof',
```

Comment:
now in development we change this summaryId property to more appropriate competingPairsId